flatdata-py/tests/__init__.py

```diff
@@ -0,0 +1,56 @@
+"""Common testing helpers & data"""
```

Comment:
So much stuff in init files... normally I would never look into those.

Reply:
That's part of what's `__init__.py` for, you import `.` and you get the common stuff everywhere if you are part of this module.